nautobot/extras/utils.py

```diff
@@ -360,6 +363,7 @@ def refresh_job_model_from_job_class(job_model_class, job_source, job_class, *,
             "grouping": job_class.grouping[:JOB_MAX_GROUPING_LENGTH],
             "name": job_class.name[:JOB_MAX_NAME_LENGTH],
             "is_job_hook_receiver": issubclass(job_class, JobHookReceiver),
+            "is_job_button_receiver": issubclass(job_class, JobButtonReceiver),
```

Comment:
Since we now have multiple subclasses for jobs, can you throw in a check for `isinstance(job_class, JobHookReceiver) and isinstance(job_class, JobButtonReceiver)` and reject the job?

Reply:
Added in latest commits.